epubmaker_postprocessing.rb

```diff
@@ -1,10 +1,11 @@
 require 'fileutils'
 require 'net/smtp'
 
-unless defined?(ENV['TRAVIS_TEST'])
+unless (ENV['TRAVIS']) == 'true'
```

Comment:
So where is the TRAVIS env variable defined? I don't see it in the yaml. Is it a system variable?